pykeepass/pykeepass.py

```diff
@@ -854,10 +945,13 @@ def create_database(
     keepass_instance.filename = filename
     keepass_instance.password = password
     keepass_instance.keyfile = keyfile
+    keepass_instance.authentication_factors = authentication_factors
+    keepass_instance._factor_data = factor_data
 
-    keepass_instance.save(transformed_key)
+    keepass_instance.save(transformed_key=transformed_key)
```

Comment:
I think this was bugged before? If I understand correctly it would pass `transformed_key` as the `filename`. So I guess nobody really used this function that way?